migrations/0002_06.08.2021-migrations.up.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE IF NOT EXISTS roles (
+   id serial PRIMARY KEY,
+   name varchar NOT NULL,
+   description varchar NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS permisions (
+   id serial PRIMARY KEY,
+   name varchar NOT NULL,
+   description varchar NOT NULL,
```

Comment:
do we need descriptions? I can't think up an examle with non-descriptive role, permission, schedule

Reply:
what about the Operator role